src/components/footer/footer.jsx

```diff
@@ -1,8 +1,77 @@
 import styles from "@/styles/components/footer/footer.module.scss";
 
+import {useTranslations} from 'next-intl';
+import Image from "next/image";
+import Link from "next/link";
+
+import React, { Component } from "react";
+import { AiOutlineInstagram } from "react-icons/ai";
+import { MdOutlineMailOutline, MdFacebook } from "react-icons/md";
+import { HiOutlinePhone } from "react-icons/hi";
+
 export default function Footer() {
+
+  const t = useTranslations('Footer');
+
   return (
-    <div> Footer yay</div>
+    <footer className={styles.footer}>
+        <div className={styles.body}>
+            {/* nav buttons */}
+            <ul className={styles.pages}>
```

Comment:
If you want to get more practice with JS, a good way to utilize the `map()` function here is to generate your `<li>` elements. You'd essentially call map to iterate through the `pages` section of the json and create a `<li>` for each of the keys.

Reply:
I'll give it a shot